CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased 3.6.1]
 ### Added
   - GEOS-only updates
+  - Throw an error if HEMCO reads netCDF data on a climatological calendar (`360_day`, `365_day`, `noleap`). These are not supported in ESMF/MAPL, so for consistency's sake, we also need to not support these when running HEMCO outside of the ESMF environment.
```

Comment:
I think this could be simplified to "Throw an error if input data calendar is not supported" This is to keep the changelog as concise as possible, preferably one-liners.

Reply:
Now fixed in commit ba24290,

Reply:
A further fix for a syntax error was made in https://github.com/geoschem/HEMCO/commit/7e20b3d0da0b77ecbd46e87af7a04c1b532442ca.